integration/nwo/template/orderer_template.go

```diff
@@ -62,6 +62,7 @@ Consensus:
   WALDir: {{ .OrdererDir Orderer }}/etcdraft/wal
   SnapDir: {{ .OrdererDir Orderer }}/etcdraft/snapshot
   EvictionSuspicion: 5s
+  Type: {{ $w.Consensus.Type }}
```

Comment:
why do we need this?

Reply:
That was added in [this commit ](https://github.com/hyperledger/fabric/pull/3781/commits/f9d8e0d2e83affffab4e29929f2666fa7e10e384)to fix the raft integration tests.